.env.sepolia

```diff
@@ -28,9 +27,10 @@ OP_NODE_P2P_BOOTNODES=enr:-Iu4QI5BdaTmfMhNKXd-vSniNRSCXk-a3m3nQbe9TCV3puGtCr6FvX
 OP_NODE_P2P_LISTEN_IP=0.0.0.0
 OP_NODE_P2P_LISTEN_TCP_PORT=9222
 OP_NODE_P2P_LISTEN_UDP_PORT=9222
+OP_NODE_P2P_STATIC=/ip4/34.34.0.167/tcp/9003/p2p/16Uiu2HAm4KjD2ig99Cy1hFMahfsUstLk6LNv5YojPLBbU8AdJVDJ
 OP_NODE_ROLLUP_CONFIG=sepolia/rollup.json
 OP_NODE_RPC_ADDR=0.0.0.0
-OP_NODE_RPC_PORT=8545
+OP_NODE_RPC_PORT=9545
```

Comment:
why are we changing the PORT?

Reply:
I'd already clarified in the PR description 😅

> Update op-node RPC port to the default value (9545), so that the README commands remain consistent for users running on Docker/CLI (from source)